.gitignore

```diff
@@ -3,3 +3,4 @@ dist/**/*
 src/config/config.json
 .env
 *.log
+settings.json
```

Comment:
?

Reply:
It wanted to add my VScode settings to the GitHub so its being ignored.

Reply:
hmm. I don't want to taint the gitignore too much. Can you make those vsc settings global so there's no need for a settings file in the workspace?